SkietbaanBE/SkietbaanBE/Migrations/20190206125741_NewMigration.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+using System.Collections.Generic;
+
+namespace SkietbaanBE.Migrations
+{
+    public partial class NewMigration : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
```

Comment:
What is the point of pushing this file with empty functions?

Reply:
someone else is working on it, i had to pull before pushing